packages/storybook-vue/config/main.ts

```diff
@@ -1,28 +1,21 @@
 import type { StorybookConfig } from '@storybook/vue3-vite';
-import path from 'path';
+import path, { dirname, join } from 'node:path';
 
 const config: StorybookConfig = {
-  features: {
-    buildStoriesJson: true,
-    storyStoreV7: true,
-  },
-  stories: ['../src/stories/**/*.stories.mdx', '../src/stories/**/*.stories.ts'],
+  stories: ['../src/stories/**/*.mdx', '../src/stories/**/*.stories.ts'],
```

Comment:
Waarom niet `.stories.mdx`, zoals het was?

Reply:
Omdat het verouderd is
- https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-related-changes
- https://storybook.js.org/docs/releases/migration-guide-from-older-version#storiesmdx-to-mdxcsf

Reply:
Duidelijk, thanks!